FilecoinFoundationWeb/filecoin-foundation
# [UXIT-1470] - Include Digest Articles

## 📝 Description

This PR includes Digest Articles for the `digest` page

- **Type:** New feature

## 📌 To-Do Before Merging

- [x] Add new assets


## 🛠️ Key Changes

- new digest content
- new assets

## 🧪 How to Test

- **Steps to Test:**
  1. go to `/digest `page
  2. check the layout, card and `/digest/[slug]` content
  3. [here are all the articles](https://docs.google.com/spreadsheets/d/1sAlLOSj0WsfRYqptzcYPgLLsMVVUOb7O7-ipvi5WoKs/edit?gid=441027424#gid=441027424) - I haven't included no.2 and no.7 as they are still not finalised


## 📸 Screenshots
![screencapture-localhost-3000-digest-2024-08-29-14_07_49](https://github.com/user-attachments/assets/2297fc93-ecad-40e8-8549-81d6a071bf6c)

![screencapture-localhost-3000-digest-letter-from-the-guest-editor-2024-08-29-14_08_02](https://github.com/user-attachments/assets/21f6842b-b41d-4d74-adee-1df5a7a82070)
